src/Service/ControllerManagerFactory.php

```diff
@@ -13,7 +13,7 @@
 use Zend\ServiceManager\FactoryInterface;
 use Zend\ServiceManager\ServiceLocatorInterface;
 
-class ControllerLoaderFactory implements FactoryInterface
+class ControllerManagerFactory implements FactoryInterface
```

Comment:
as https://github.com/zendframework/zend-authentication/pull/2#issuecomment-110043990 , ControllerLoader actually removed for 3.0